db/schema.rb

```diff
@@ -190,7 +190,7 @@
 
   create_table "declaration_states", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "participant_declaration_id", null: false
-    t.string "state", default: "submitted", null: false
+    t.string "state", default: "submitted"
```

Comment:
is this supposed to be here? there's no corresponding migration

Reply:
Now updated